sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/TableResourcesOperationsTests.cs

```diff
@@ -14,103 +14,147 @@ public class TableResourcesOperationsTests : CosmosDBManagementClientBase
-            Assert.AreEqual(defaultThroughput, throughputSettingsGetResults1.Resource.Throughput);
+        [TestCase, Order(2)]
+        public async Task TableMigrateToAutoscaleTest()
+        {
+            ThroughputSettingsGetResults throughputSettingsGetResults = await WaitForCompletionAsync(
+                await CosmosDBManagementClient.TableResources.StartMigrateTableToAutoscaleAsync(resourceGroupName, databaseAccountName, tableName));
+            Assert.IsNotNull(throughputSettingsGetResults);
+            Assert.IsNotNull(throughputSettingsGetResults.Resource.AutoscaleSettings);
+            Assert.AreEqual(defaultMaxThroughput, throughputSettingsGetResults.Resource.AutoscaleSettings.MaxThroughput);
+            Assert.AreEqual(defaultThroughput, throughputSettingsGetResults.Resource.Throughput);
+        }
 
-            ThroughputSettingsGetResults throughputSettingsGetResults2 =
-                await WaitForCompletionAsync(await CosmosDBManagementClient.TableResources.StartMigrateTableToManualThroughputAsync(resourceGroupName, databaseAccountName, tableName));
-            Assert.IsNull(throughputSettingsGetResults2.Resource.AutoscaleSettings);
-            Assert.AreEqual(defaultMaxThroughput, throughputSettingsGetResults2.Resource.Throughput);
+        [TestCase, Order(3)]
+        public async Task TableMigrateToManualThroughputTest()
+        {
+            ThroughputSettingsGetResults throughputSettingsGetResults = await WaitForCompletionAsync(
+                await CosmosDBManagementClient.TableResources.StartMigrateTableToManualThroughputAsync(resourceGroupName, databaseAccountName, tableName));
+            Assert.IsNotNull(throughputSettingsGetResults);
+            Assert.IsNull(throughputSettingsGetResults.Resource.AutoscaleSettings);
+            Assert.AreEqual(defaultMaxThroughput, throughputSettingsGetResults.Resource.Throughput);
+        }
 
-            var throughputSettingsUpdateParameters = new ThroughputSettingsUpdateParameters(new ThroughputSettingsResource(defaultThroughput, null, null, null));
-            ThroughputSettingsGetResults throughputSettingsGetResults = (
-                await WaitForCompletionAsync(
-                    await CosmosDBManagementClient.TableResources.StartUpdateTableThroughputAsync(resourceGroupName, databaseAccountName, tableName, throughputSettingsUpdateParameters))).Value;
+        [TestCase, Order(4)]
+        public async Task TableUpdateThroughputTest()
+        {
+            ThroughputSettingsGetResults throughputSettingsGetResults = await WaitForCompletionAsync(
+               await CosmosDBManagementClient.TableResources.StartUpdateTableThroughputAsync(
+                   resourceGroupName,
+                   databaseAccountName,
+                   tableName,
+                   new ThroughputSettingsUpdateParameters(new ThroughputSettingsResource(defaultThroughput, null, null, null))));
```

Comment:
Isn't this tables throughput already the default (defaultThroughput) per TableMigrateToAutoscaleTest line 109?